cmd/env.go

```diff
@@ -361,3 +322,32 @@ func commonEnvFlags(flags *pflag.FlagSet) (server, namespace, context string, er
 
 	return server, namespace, context, nil
 }
+
+func resolveEnvFlags(flags *pflag.FlagSet) (server, namespace string, err error) {
```

Comment:
Named returns coupled with a naked return are hard to reason about in functions longer than a few lines. In this case, these aren't needed since you are returning concrete values below.